drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/strategies/DataManagementStrategy.java

```diff
@@ -26,9 +32,15 @@
  */
 public interface DataManagementStrategy {
 
+    Map<String, Class> SIMPLE_CLASSES_MAP = Collections.unmodifiableMap(Stream.of(
+            new AbstractMap.SimpleEntry<>(Boolean.class.getSimpleName(), Boolean.class),
+            new AbstractMap.SimpleEntry<>(Double.class.getSimpleName(), Double.class),
+            new AbstractMap.SimpleEntry<>(Integer.class.getSimpleName(), Integer.class),
+            new AbstractMap.SimpleEntry<>(Number.class.getSimpleName(), Number.class),
+            new AbstractMap.SimpleEntry<>(String.class.getSimpleName(), String.class)).
```

Comment:
This list seems to me as not complete list of simple types, have you considered reuse(extend and reuse) existing class that collects simple data types?

https://github.com/kiegroup/kie-soup/blob/master/kie-soup-project-datamodel/kie-soup-project-datamodel-api/src/main/java/org/kie/soup/project/datamodel/oracle/DataType.java

Reply:
This list reflect what we are currently supporting. It is a sublist of what the user can import from the "Import" tab. Inside that tab, there are also List/ArrayList/Collection - that does not makes sense for us.

Reply:
Ok, list of Types on Imports tab comes from this line [1] and is shown to the user here [2]. I still think we should not re-declare list of something existing. However have no resources investigate more neither I am going to approve or decline, wanted just discuss this topic.

[1]
https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-widgets/kie-wb-config-resource-widget/src/main/java/org/kie/workbench/common/widgets/configresource/client/widget/bound/ImportsWidgetPresenter.java#L83

[2]
https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-widgets/kie-wb-config-resource-widget/src/main/java/org/kie/workbench/common/widgets/configresource/client/widget/bound/ImportsWidgetViewImpl.java#L227